jooby-netty/src/main/java/org/jooby/internal/netty/NettyResponse.java

```diff
@@ -184,11 +189,28 @@ public void send(final FileChannel channel, final long offset, final long count)
     rsp.headers().set(headers);
     ChannelHandlerContext ctx = this.ctx;
     ctx.channel().attr(NettyRequest.NEED_FLUSH).set(false);
+
+    // add chunker
+    ChannelPipeline pipeline = ctx.pipeline();
+    if (pipeline.get("chunker") == null) {
```

Comment:
That's true, though it won't hurt to have it there anyway. I even thought about moving it away here and adding it to NettyPipeline, but that's more of a design choice. I'll leave that to you :) Feel free, to only add it when there's SSL involved.